config/Migrations/20200610160031_AddPublicplus.php

```diff
@@ -0,0 +1,28 @@
+<?php
+use Migrations\AbstractMigration;
+
+class AddPublicplus extends AbstractMigration
+{
+    /**
+     * Change Method.
+     *
+     * More information on this method is available here:
+     * http://docs.phinx.org/en/latest/migrations.html#the-change-method
+     * @return void
+     */
+    public function change()
+    {
+    $table = $this->table('sentences_lists');
+
+    $table->changeColumn('visibility', 'enum', [
+        'after' => 'modified',
+        'default' => 'unlisted',
+        'null' => false,
+        'values' =>[
+            'private', 
+            'unlisted', 
+            'public', 
+            'public+'] 
+        ])->save();        
```

Comment:
The [Phinx docs](https://book.cakephp.org/phinx/0/en/migrations.html#the-change-method) recommend using `update()` instead of `save()` when changing a column inside the `change` method.

And I'm not sure whether `public+` is really a good name for the option. But @trang and @jiru will have the final call about this.